app/controllers/user.js

```diff
@@ -1,21 +1,18 @@
 const validations = require('../util');
 const userService = require('../services/users');
-const errors = require('../errors');
 const logger = require('../logger');
 
 exports.createUser = (req, res, next) => {
-  const { password, email } = req.body;
-  if (validations.paramsValidation(password, email)) {
-    return next(errors.invalidParameters('The email or password does not meet the conditions.'));
-  }
+  const user = req.body;
   return validations
-    .passwordEncryption(req.body)
+    .passwordEncryption(user)
     .then(userService.userRegister)
     .then(response => {
-      logger.info(`User ${response.firstName} ${response.lastName} created successfully`);
+      logger.info(`User ${user.firstName} ${user.lastName} created successfully`);
       return res.status(201).send({
-        firstName: response.firstName,
-        lastName: response.lastName
+        firstName: user.firstName,
+        lastName: user.lastName,
+        message: `User has been created successfully with ID ${response.id}`
```

Comment:
Why do you send a message in the response? You dont do this in APIs Rest, you just send the data with a status code